lograge.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.description = "Tame Rails' multi-line logging into a single line per request"
   s.license     = 'MIT'
 
-  s.files = `git ls-files lib`.split("\n")
+  s.files = `git ls-files lib license.txt`.split("\n")
```

Comment:
This is case-sensitive, so it'll need to be `LICENSE.txt`.

Reply:
```suggestion
  s.files = `git ls-files lib LICENSE.txt`.split("\n")
```

Reply:
Thanks, done.